src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/Browser/WasmTestBrowserCommand.cs

```diff
@@ -105,8 +109,37 @@ protected override async Task<ExitCode> InvokeInternal(ILogger logger)
         }
 
         private (DriverService, IWebDriver) GetChromeDriver(ILogger logger)
+            => GetChromiumDriver<ChromeOptions, ChromeDriver, ChromeDriverService>(
+                        "chromedriver",
+                        options => ChromeDriverService.CreateDefaultService(),
+                        logger);
+
+        private (DriverService, IWebDriver) GetEdgeDriver(ILogger logger)
+        {
+            string driverName = "msedgedriver";
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                driverName = "msedgedriver.exe";
```

Comment:
I wonder if this is because it's looking for the legacy EdgeHTML driver, the `options.UseChromium` parameter seems to be false by default: https://alexanderontesting.com/2020/05/25/new-tricks-with-chromium-edge-and-msedgedriver-exe/

Can you try whether setting that option causes it to look for the correct driver name?